redis_sessions/alt_settings.py

```diff
@@ -0,0 +1,15 @@
+from settings import *
+
+TAG = "alt_"
+# this is the alternative redis store instance configs
+SESSION_REDIS = getattr(settings, 'ALT_SESSION_REDIS', {})
```

Comment:
Not sure what implication this actually has on the migration ... In my thinking, as this two Redis instances are identical, what should be changing is the host. Do I have to redefine the whole dictionary just to override the host?

Reply:
Same question I had

Reply:
Which then begs the question as to whether we need this settings file

Reply:
Also, I'm not sure the alternate settings file approach is the best